Improve docs

ai opened the pull request:
First attempt for docs improvments:

1. Remove `nanocolors` references since we will close the project.
2. Start from clear description what library does.
3. Explain why I care about color library size
4. Add direct comparison  with `chalk` because direct comparisons is very common for US culture (but yes, it is prohibited in Russian culture, but I think US culture is better here)
5. Use `picocolors`’s benchmarks (I took performance results from CI).

TrySound commented on README.md:
```diff
@@ -81,43 +149,25 @@ The library provides additional utilities to ensure the best results for the tas
+## Replacing `chalk`
 
-`nanocolors` [benchmark](https://github.com/ai/nanocolors/tree/main/test):
+1. Replace import and use named exports:
```
Not named anymore?

ai replied:
We improved benchmarks a lot. `picocolors`’s benchmarks are now much closer to the reality.

We found that text size affects benchmarks and tests like `red('test')` doesn’t show the real performance. Now `complex` benchmark is try to combine different use cases to be closer to the reality.

ai replied:
`nanocolors` will be mentioned in Prior Art. We do not need to put it to benchmark results.

TrySound replied:
No, I mean picocolors doesn't have named exports anymore, right?
> Replace import and use named exports:

ai replied:
Fixed 😅

TrySound commented on package.json:
```diff
@@ -7,7 +7,7 @@
   },
   "sideEffects": false,
-  "description": "The tiniest and the fastest coloring library ever",
+  "description": "14s smaller than chalk and 2x faster",
```
This will appear in npm search and may look confusing as people may not be familiar with chalk
![image](https://user-images.githubusercontent.com/5635476/136019101-04cdce02-a78d-4ad5-be5d-dc69e2f1bc93.png)

ai replied:
Yeap, we can revert it back

ai replied:
Fixed

ai commented:
> Regarding "brief description": I think there has to be a single line, that is used as the repo description, package.json description and the first line of README. Here is my take: "the tiniest and the fastest library for terminal output formatting with ANSI colors".

Done. I changed features list (I keep “big users” reason above because it is really what people care about for utility dependency).

I join `chalk` mentions in the single line and changed description.
